org.eu.wabarc.wayback.yml

```diff
@@ -23,11 +23,6 @@ modules:
         sha256: bca11c1ca33cb03f631efa4d0b304852826ace00580963699520d32e52e7add9
         only-arches:
           - x86_64
-      - type: file
-        url: https://github.com/wabarc/wayback/releases/download/v0.18.1/wayback-linux-arm64-0.18.1.tar.gz
-        sha256: 462087adc88f920a06b63d7e127bf822502d7e7c734d0c1c46b5390077276fbf
-        only-arches:
-          - aarch
```

Comment:
this is not what i meant by "invalid arch".

The proper arch id for ARM v8 64-bits is `aarch64`.